iothub_client/inc/internal/iothub_internal_consts.h

```diff
@@ -10,6 +10,9 @@ extern "C"
 #endif
 
     static const char* IOTHUB_API_VERSION = "2017-11-08-preview";
```

Comment:
Ugh...why is this still preview?

Reply:
Ugh indeed.  We're having side discussions RE getting latest updates from hub better.  I'll get going again.

This particular one I was planning on updating via #1539, the json array in twin one.  Since it's in progress already and tight deadline on this change, I'd wanted to not touch it here.